### PR TITLE
Support proxy

### PR DESCRIPTION
**Pull request by Poil**

### CODE REVIEW

**Comment by Poil** [comment]
Done if it's OK for you :)

File: docker/kernel.sls [hunk 1]
```diff
@@ -25,5 +25,8 @@ docker-dependencies-kernel:
     - require_in:
       - pkg: docker package
     - onlyif: dpkg --compare-versions {{ grains["kernelrelease"] }} lt 3.8
+    {%- if docker.proxy %}
+    - proxy: {{ docker.proxy }}
+    {%- endif %}
```
Comment (javierbertoli) [inline docker/kernel.sls]: I think this one is wrong? `pkg.installed` does not support `proxy`, am I right?

Comment (Poil) [reply docker/kernel.sls]: My bad ... fixing it

**Comment by Poil** [comment]
Removed, but I did'nt understand your first comment

**Pull request merged**